Question/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "question",
+  "version": "1.0.0",
+  "description": "",
+  "main": "QuestionServer.js",
+  "scripts": {
+    "dev": "nodemon QuestionServer.js",
+    "start": "nodemon QuestionServer.js"
```

Comment:
```suggestion
    "start": "node QuestionServer.js"
```

Reply:
I think you can use port 3001 since nothing is using it, but no issues using 6001 as well.

Reply:
updated

Reply:
If so, should User api revert back to using relative since the paths complexity is around the same?

Reply:
Im good with both!